packages/eslint-config-terra/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "eslint-config-terra",
+  "version": "3.10.0",
+  "description": "Terra Eslint Config",
+  "main": "eslint.config.js",
```

Comment:
Lost the link to the documentation, but a `directory` key can be defined here for each package in a monorepo. We've started adding it into packages in terra-core.

```
"directory": "packages/eslint-config-terra"
````

Reply:
Here's the doc link: https://docs.npmjs.com/files/package.json#repository

Reply:
https://github.com/cerner/terra-toolkit/pull/402/commits/d3e6de1d2d39e3db1a4208e58900a71f34df011e